operations/workspace/deployment/pkg/step/installgitpod.go

```diff
@@ -17,18 +21,39 @@ const (
 	DefaultGitpodInstallationScript = "dev/build-ws-cluster/install-gitpod.sh"
 )
 
-func InstallGitpod(context *common.ProjectContext, gitpodContext *common.GitpodContext, cluster *common.WorkspaceCluster) error {
-	err := runner.ShellRunWithDefaultConfig(DefaultGitpodInstallationScript, []string{"-g", context.Id, "-l", cluster.Region, "-n", cluster.Name, "-a", getValuesFilesArgString(gitpodContext)})
-	return err
+func InstallGitpod(context *common.Context, cluster *common.WorkspaceCluster) error {
+	if context.Overrides.DryRun {
+		log.Log.Infof("dry run not supported for gitpod installation. skipping...")
+		return nil
+	}
+	credFileEnvVar := fmt.Sprintf("GOOGLE_APPLICATION_CREDENTIALS=%s", context.Project.GCPSACredFile)
+	if _, err := os.Stat(context.Project.GCPSACredFile); errors.Is(err, os.ErrNotExist) {
+		// reset this to empty string so that we can fallback to default
+		// gcloud context. This is useful in local development and execution
+		// scenarios
+		credFileEnvVar = ""
+	}
+
+	// add versions manifest
+	valuesFiles := append(cluster.ValuesFiles, context.Gitpod.VersionsManifestFilePath)
```

Comment:
Passing this as a values file did not work. As the content should be under the key, `gitpod`.